.misc/check_unsupported.sh

```diff
@@ -0,0 +1,23 @@
+#!/usr/bin/env bash
+
+set +e
+
+# Enable capturing the non-zero exit status of coverage instead of tee
+set -o pipefail
+
+coverage run setup.py install | tee setup.log
+
+retval=$?
+
+if [[ $retval != 4 ]]; then
+  echo "Unexpected error code $?"
+  if [[ $retval == 0 ]]; then
```

Comment:
Sort of.
We expect error code 4 (per https://github.com/coala/coala/blob/master/coalib/__init__.py#L27)
So 4 means the script exists without error (0).

If we see 0, then we cant exit without error (0), so we map 0 to a different value.